cdap-ui/app/features/tracker/directives/lineage/lineage.html

```diff
@@ -0,0 +1,131 @@
+         ng-if="node.isLeftEdge && node.showNavigation"
+         ng-mouseenter="Lineage.mouseEnterNavigation(node)"
+         ng-mouseleave="node.showNavigation = false">
+      <span class="fa fa-caret-left" ng-click="Lineage.navigationClick($event, node)"></span>
+    </div>
+    <div class="navigation-control right"
+         ng-if="node.isRightEdge && node.showNavigation"
+         ng-mouseenter="Lineage.mouseEnterNavigation(node)"
+         ng-mouseleave="node.showNavigation = false">
+      <span class="fa fa-caret-right" ng-click="Lineage.navigationClick($event, node)"></span>
+    </div>
+
+    <h4 class="entity-name">{{ Lineage.uniqueNodes[node.uniqueNodeId].label }}</h4>
+    <div class="entity-type">
+      <span class="{{ Lineage.uniqueNodes[node.uniqueNodeId].icon }}"></span>
+      <span>{{ Lineage.uniqueNodes[node.uniqueNodeId].displayType }}</span>
+    </div>
+
+
+    <!-- PROGRAM POPOVER -->
+    <div class="tracker-program-popover" ng-if="node.showPopover" ng-click="Lineage.preventPropagation($event)">
+      <div class="tracker-popover-content">
+        <div class="top-bar">
+          <div class="row">
+
+            <div class="col-xs-8">
+              <div class="entity-detail">
+                <h4>{{ Lineage.uniqueNodes[node.uniqueNodeId].label }}</h4>
+                <div class="entity-type">
+                  <span class="{{ contentData.icon }}"></span>
+                  <span>{{ Lineage.uniqueNodes[node.uniqueNodeId].displayType }}</span>
+                </div>
+              </div>
+            </div>
+
+            <div class="col-xs-4">
+              <div class="runs-controller">
+                <h5>Runs</h5>
+                <div class="runs-navigation">
+                  <span class="fa fa-caret-left" ng-click="Lineage.prevRun($event, node)"></span>
+                  <span>{{ node.popover.activeRunIndex + 1 }}/{{ Lineage.uniqueNodes[node.uniqueNodeId].runs.length }}</span>
+                  <span class="fa fa-caret-right" ng-click="Lineage.nextRun($event, node)"></span>
+                </div>
+              </div>
+            </div>
+
+            <div class="close-button">
+              <span class="fa fa-close" ng-click="Lineage.closePopover($event, node)"></span>
+            </div>
+          </div>
+        </div>
+
+        <div class="info-container">
+          <div class="row">
+            <div class="col-xs-12">
+              <div class="inside-column">
+                <h5>Run ID</h5>
+                <div>
+                  <span>{{ node.popover.activeRunId }}</span>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="row">
+            <div class="run-detail inside-column">
+              <div class="col-xs-5">
+                <h5>
+                  Started
+                </h5>
+                <div>
+                  <span> {{ node.popover.runInfo.start * 1000 | amDateFormat: 'ddd,  MM/DD/YY h:mm:ss a' }} </span>
```

Comment:
For `runInfo.duration` (below) you do the `* 1000` in the controller, but for this, you do it in the view.
Not a big deal, but I think it would be cleaner if both did it in the same place.